src/api.v2/routes/gem2s.js

```diff
@@ -0,0 +1,13 @@
+const { runGem2s, handleResponse } = require('../controllers/gem2sController');
+
+const { expressAuthorizationMiddleware } = require('../middlewares/authMiddlewares');
+
+module.exports = {
+  'gem2s#run': [
+    expressAuthorizationMiddleware,
+    (req, res, next) => runGem2s(req, res).catch(next),
+  ],
+  'gem2s#response': (req, res, next) => {
+    handleResponse(req, res).catch(next);
```

Comment:
do you know why we don't have any authentication / authorization here? I hadn't realized.

Reply:
I checked, the from what I say what we have of authentication is in `parseSNSMessage`. 
It's also using something called [sns-validator](https://www.npmjs.com/package/sns-validator)